content/contact.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Contact"
+date: 2017-07-03T14:28:34-04:00
+draft: false
+mainpage: false
+projectpage: false
+projectslug: ""
+projectname: ""
+listdescription: ""
+---
+
+### Contact
+If you would like to reach out to the NASA WorldWind team, please try the following:
+
+---
+#### NASA Program Management
```

Comment:
I think Patrick wanted this to read NASA Administrative Contact, like it did on the current site at https://worldwind.arc.nasa.gov.